test/Motion-test.js

```diff
@@ -548,9 +549,9 @@ describe('Motion', () => {
       {a: 350.02047519790233},
     ]);
     mockRaf.step(999);
-    expect(count.length).toBe(85);
+    expect(count.length).toBe(86);
```

Comment:
Why changing this ?

Reply:
Because component is going to have one more update I assume due to switching to componentDidUpdate